stable/cluster-autoscaler/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 3.1.0
+version: 3.1.1
```

Since this change set adds a feature, let's bump the minor version.
```suggestion
version: 3.2.0
```

---

bumped.